### ISSUE TITLE
[A11y] Work Show Page-Multiple - Link Purpose

### ISSUE THREAD

**Issue by jechols** [OregonDigital/OD2#2174]
### Operating System

Windows

### Browser

Firefox

### Affected page

Work Show

### Admin user?

Yes

### Issue Description

https://staging.oregondigital.org/concern/generics/df727084j
https://staging.oregondigital.org/concern/documents/df727057n

All links to child objects and related items have the same accessible name, "Danforth Cooke & Co., Patterson, NJ. Engine purchases by OSN...". These need to have a unique name since they point to different pages. Ideally these works would start with the page or sequence number so it's immediately obvious which page a link goes to, and the links could continue to be abbreviated since the critical differentiation would remain.

### Suggested Fix

If we're abbreviating these links in code, we should stop, because that's going to cause systemic problems. If this is something in our metadata, we have a bigger problem and a more difficult fix.

If at all possible, we should have the page number (the piece of text that actually differentiates the links) before the rest of the text.

### Can anybody QA or does this need an expert?

A11y familiarity recommended

### QA Steps

Make sure the links all have a different accessible name

**Comment by straleyb** [comment]
These names should be unique
<img width="1675" alt="Screen Shot 2022-02-07 at 11 44 08 AM" src="https://user-images.githubusercontent.com/6424683/152860215-a5640f84-2e45-4a54-a9ad-1887b8783e77.png">
<img width="946" alt="Screen Shot 2022-02-07 at 11 44 48 AM" src="https://user-images.githubusercontent.com/6424683/152860330-838b6bea-b38b-4a2e-a504-237c1807040a.png">

**Comment by azle** [comment]
Updated Wave and Accessibility Insights are reporting errors 12/26/2024

Please review errors on WAVE
<img width="372" alt="Screenshot 2024-12-26 at 7 49 59 AM" src="https://github.com/user-attachments/assets/411f5657-6397-44cd-80e1-a9e56fb27d78" />

Once errors are corrected on WAVE needs to do a pass in Accessibility Insights.
